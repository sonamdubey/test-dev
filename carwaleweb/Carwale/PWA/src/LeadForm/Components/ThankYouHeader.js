import React, { PureComponent } from "react";
import PropTypes from "prop-types";

export class ThankYouHeader extends PureComponent {
  render() {
    const containerClass = `thankyou-header ${this.props.className}`;

    return (
      <div id="postsubmitmsg" className={containerClass}>
        <p className="thankyou-header__title">Thank You {this.props.name}!</p>
        <p className="thankyou-header__subtitle">
          A car consultant would get in touch with you shortly with assistance
          on your purchase.
        </p>
      </div>
    );
  }
}

ThankYouHeader.propTypes = {
  name: PropTypes.string
};

export default ThankYouHeader;
