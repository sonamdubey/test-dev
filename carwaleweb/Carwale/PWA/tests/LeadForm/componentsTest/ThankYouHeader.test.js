import React from "react";
import ThankYouHeader from "../../../src/LeadForm/Components/ThankYouHeader";

test("Component ThankYouHeader Snapshot RenderedTree", () => {
  const props = {
    name: "test"
  };

  let wrapper = shallow(<ThankYouHeader {...props} />);
  expect(wrapper).toMatchSnapshot();
});
