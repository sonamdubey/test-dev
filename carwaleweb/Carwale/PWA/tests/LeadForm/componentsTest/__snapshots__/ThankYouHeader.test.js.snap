// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ThankYouHeader Snapshot RenderedTree 1`] = `
<div
  className="thankyou-header undefined"
  id="postsubmitmsg"
>
  <p
    className="thankyou-header__title"
  >
    Thank You 
    test
    !
  </p>
  <p
    className="thankyou-header__subtitle"
  >
    A car consultant would get in touch with you shortly with assistance on your purchase.
  </p>
</div>
`;
